src/hotspot/share/classfile/classFileParser.cpp

```diff
@@ -3893,9 +3893,16 @@ void ClassFileParser::parse_classfile_attributes(const ClassFileStream* const cf
                          class_info_index, CHECK);
           _nest_host = class_info_index;
 
-        } else if (_major_version >= JAVA_15_VERSION) {
-          // Check for PermittedSubclasses tag
-          if (tag == vmSymbols::tag_permitted_subclasses()) {
+        } else if (_major_version >= JAVA_16_VERSION) {
```

Comment:
I don't understand why you changed it from >= 15 to >= 16.

Reply:
It was changed to 16 because, starting with 16, Records is a new non-preview feature.   So, the Record attribute is for class file version 60 or later.  60 is the major class file version for JDK-16.

Sealed types are still preview but require class file version 60.65535.   Preview features must always have the latest major class file version (and minor version of 655353).

Reply:
Okay, thank you for the explanation about preview features.  Looks good to me.